pkg/route/registry/route/rest_test.go

```diff
@@ -105,7 +114,11 @@ func TestCreateRouteOK(t *testing.T) {
 
 func TestGetRouteError(t *testing.T) {
 	mockRegistry := test.NewRouteRegistry()
```

Comment:
Its more than possible I am using it in a wonky manner. Is there a git ```pre-commit``` hook somewhere I can use?